framework/utils/src/commonTest/kotlin/com/cramsan/framework/utils/loginvalidation/SignUpValidationTests.kt

```diff
@@ -190,7 +190,19 @@ class SignUpValidationTests {
         // Act
         val result = validatePassword(password)
         // Assert
-        assertTrue(result.isEmpty())
+        assertTrue { result.contains("Password must contain at least one symbol.") }
```

Comment:
This test should check if a valid password is valid and returns an empty list. Also, update this to say `validatePassword__has_valid_password_and_returns_empty_list()`

On line 171 `validPassword_has_invalid_password_and_returns_list` should check a symbol is not present. We should update the tests to check the size we expect. 
Also, update the title of the method to ``validatePassword_has_invalid_password_and_returns_list`